lib/_stream_writable.js

```diff
@@ -60,6 +60,8 @@ const {
 
 const { errorOrDestroy } = destroyImpl;
 
+const afterWriteTickInfo = Symbol('afterWriteTickInfo');
```

Comment:
I think we prefix symbols with `k` i.e. `kAfterWriteTickInfo`

Reply:
Done